### DISCUSSION TITLE
blog/knowing-enough-about-moe/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#11] [Announcements]
# blog/knowing-enough-about-moe/

In a previous blogpost, I made a simple observation about GPT-4 from a paper I had incidentally read on a whim. After finishing the post, I realised I didn’t actually ever figure out how token dropping could occur; only learning a black-box rule that it could occur in batched MoE inference for reasons.
This post is here to fix that – to collect enough info from important MoE papers (and alleged GPT-4 leaks) to explain the full mechanism of token dropping.

https://152334h.github.io/blog/knowing-enough-about-moe/

**Comment by kevinamiri** [comment]
I do not think that GPT-4 has anything more than llama2 or mistral, The rest is about fine-tuning. I have never seen one example opensurce model that uses the basic principles (rules that OpenAI clearly wrote in its papers over the last 4 years). To me, this is a mystery why no one cares. Companies are investing/wasting so much money to create a model that passes outdated evaluations dating back to the years of GPT-2. Most of the research aims at writing papers like master's theses or something to put in their CV. Why try to catch up to GPT-4 instead of something better? 

I noticed your profile on HF, and recall that you were the one who did the best job on tortoise.
